maven-archetype-plugin/pom.xml

```diff
@@ -172,6 +172,154 @@
   </dependencies>
 
   <build>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-plugin-report-plugin</artifactId>
+          <configuration>
+
+            <requirementsHistories>
+
+              <requirementsHistory>
```

Comment:
let's drop the 1.0 and 2.0 alpha versions.

Reply:
i'll fix it